### ISSUE TITLE
New option: hide-title-and-hints

### ISSUE THREAD

**Issue by sudo-nice** [akrennmair/newsbeuter#345]
Please, consider to add such an option for hiding the titlebar and the hintbar (that ones, we alreade can swap with `swap-title-and-hints` option). This could increase the space of valuable information for whole two rows! :wink:

**Comment by Minoru** [comment]
This should probably be two separate options, as I can well imagine an experienced user who doesn't need hints but would still like to see the title of the feed/article they are reading.

**Comment by Minoru** [comment]
Indeed, there's `show-keymap-hint` already, so we only have to add `show-titlebar` or something.
